runtime/testrt.c

```diff
@@ -35,6 +35,11 @@ static TaggedPtr copySmallString(TaggedPtr tp) {
     return copy;
 }
 
+int64_t stringLen(TaggedPtr p) {
```

Comment:
using `_Bstring__length` didn't work, got `ld: symbol(s) not found` but only with `-O2`.

Reply:
Maybe need to link in balrt_inline.bc

Reply:
It is already linked, maybe since it's a `.bc` file it's not getting picked up.

https://github.com/ballerina-platform/nballerina/blob/f637cf79e0a521272c5ff74b98b4575387b77ee1/runtime/Makefile#L24